### ISSUE TITLE
Conditional Access with "Require app protection policy"

### ISSUE THREAD

**Issue by lartisimi** [Azure-Samples/active-directory-xamarin-native-v2#175]
Hello Team,

we are developing a Xamarin-app using Microsoft Authenticator as a broker app and Intune MAM silent enrollment. 
Additionally, we would like to apply a conditional access policy. Although the device-related conditional access works fine, we still have trouble with applying "Require app protection policy". While trying to sign in for the first time we get blocked by the conditional access because the Intune MAM policy is not applied yet.  

We have already checked this documentation for Swift/Objective-C https://docs.microsoft.com/en-us/mem/intune/developer/app-sdk-ios#msaladal . Does Microsoft have any documentation to resolve this issue for Xamarin apps?

**Comment by jmprieur** [comment]
@maliksahil @SameerK-MSFT @trwalke : can one of you please answer the customer?

**Comment by SameerK-MSFT** [comment]
@lartisimi does it happen only for the first time?

**Comment by mafrank1** [comment]
@SameerK-MSFT No, do to the conditional access related login error occurs before the Intune MAM Policy is applied. Therefore the login is permanently not working.
